core/src/main/java/io/grpc/internal/OobChannel.java

```diff
@@ -72,8 +74,9 @@ public ClientTransport get(PickSubchannelArgs args) {
     }
 
     @Override
-    public DelayedClientTransport getDelayedTransport() {
-      return delayedTransport;
+    public <ReqT> RetriableStream<ReqT> newRetriableStream(MethodDescriptor<ReqT, ?> method,
+        CallOptions callOptions, Metadata headers, Context context) {
+      return null;
```

Comment:
Not throws? And comment saying why it isn't implemented (like how it's impossible to enable)?

Reply:
Throwed.